packages/pn-commons/src/components/Notifications/NotificationsDataSwitch.tsx

```diff
@@ -1,72 +1,121 @@
-import { Typography } from '@mui/material';
-
+import { Box, Typography } from '@mui/material';
+import ChevronRightIcon from '@mui/icons-material/ChevronRight';
+import { ButtonNaked } from '@pagopa/mui-italia';
 import { useIsMobile } from '../../hooks';
 import { Notification, NotificationColumnData, Row } from '../../models';
 import { formatDate, getNotificationStatusInfos } from '../../utility';
+import { getLocalizedOrDefaultLabel } from '../../utility/localization.utility';
 import NewNotificationBadge, { isNewNotification } from './NewNotificationBadge';
 import StatusTooltip from './StatusTooltip';
 
-const NotificationStatusChip: React.FC<{
-  data: Row<Notification>;
-}> = ({ data }) => {
+const NotificationStatusChip: React.FC<{ data: Row<Notification> }> = ({ data }) => {
   const { label, tooltip, color } = getNotificationStatusInfos(data.notificationStatus, {
     recipients: data.recipients,
   });
   return <StatusTooltip label={label} tooltip={tooltip} color={color} />;
 };
 
+const SentAt: React.FC<{ data: Row<Notification>; isMobile: boolean }> = ({ data, isMobile }) => {
```

Comment:
Why do we need to create these components?

Reply:
for complexity and order all component